lib/features/auth/view/widget/circular_button.dart

```diff
@@ -33,9 +33,9 @@ class CircularButton extends StatelessWidget {
         ElevatedButton(
           onPressed: () {
             if (formKey != null) {
-              if (formKey?.currentState?.validate() ?? false) {
+              if (formKey!.currentState?.validate() ?? false) {
                 debugPrint('Form is valid!');
-                handelSubmit!();
+                handelSubmit != null ? handelSubmit!() : null;
```

Comment:
as you checked that handleSubmit is not null, you don't need '!' when calling the function

Reply:
Because I defined ```handelSubmit``` as ```Function?```  so even I added a check still I had to put '!' to tell the compiler that I am sure that it won't be null.

Reply:
ok